cypress/support/commands/commonCommands.js

```diff
@@ -55,3 +54,10 @@ Cypress.Commands.add('loadFAQArticle', () => {
     .first()
     .click();
 });
+
+Cypress.Commands.add('waitForFaqsBasedOnBookingStatusToLoad', () => {
+  cy
```

Comment:
one more thing - should this command also verify that _some_ faqs are loaded?

Reply:
you've got a point, as the function name says, that the faqs based on context are loaded. however it also works without that, because the commands used after, wait for yielding the specific DOM element, but it seems more clear if we add the verification here. and of course it eliminates confusion. 😃

Reply:
great, with one note 

`cy.get('[data-cy=faq-categories]').should('exist');` - this should be first (to ensure some faqs are there, then check that "About Kiwi.com" doesn't exist)

Reply:
Ah ok, then we were speaking about different things. 😄 I've added it here, to ensure that FAQs based on booking status are loaded, as the command name implies (which means that there are FAQs, but none of them is "About Kiwi.com"). 
What about, if we have both checks, for the inital load and for the ones depending on the context? 🤔

Reply:
my issue is all about to ensure that

1) some faqs are loaded (no loading or error is there)
2) none of the faqs is "about kiwi.com"

in order to ensure this, it should be done in this order, that's all

Reply:
yepp, I understand. what I meant, is:

1) some faqs are loaded (no loading or error is there)
2) none of the faqs is "about kiwi.com"
3) some faqs are loaded (based on booking status, no loading or error is there)

what do you think about this?

Reply:
@jaroslav-kubicek order fixed